tests/selenium/conf.js

```diff
@@ -6,7 +6,7 @@ const config = {
     retries: 4
   },
   specs: ['spec/*.js'],
-  exclude: ['spec/quickstarts-spec.js'],
+  // exclude: ['spec/quickstarts-spec.js'],
```

Comment:
Nit, remove

Reply:
Ugh... I had removed it, but didn't hit save